src/bases2fastq/config.vsh.yaml

```diff
@@ -0,0 +1,204 @@
+        type: integer
+        min: 1
+        description: |
+          Run parameters override, I2 cycles
+      - name: "--include_tile"
+        alternatives: [-i]
+        type: string
+        multiple: true
+        description: |
+          Regex matching tile names to include. This flag
+          can be specified multiple times. (e.g. L1.*C0[23]S.)
+      - name: "--kit_configuration"
+        type: string
+        description: |
+          Run parameters override, kit configuration.
+      - name: "--legacy_fastq"
+        type: boolean_true
+        description: |
+          Legacy naming for FASTQ files (e.g. SampleName_S1_L001_R1_001.fastq.gz)
+      - name: "--log_level"
+        type: string
+        alternatives: [-l]
+        choices: ["DEBUG", "INFO", "WARNING", "ERROR"]
+        description: |
+          Severity level for logging.
+        example: INFO
+      - name: "--no_error_on_invalid"
+        type: boolean_true
+        description: |
+          Skip invalid files and continue execution. Overridden by --strict options
+      - name:  "--no_projects"
+        type: boolean_true
+        description: |
+          Disable project directories
+      - name: "--num_unassigned"
+        type: integer
+        min: 0
+        max: 1000
+        example: 30
+        description: |
+          Max Number of unassigned sequences to report.
+      - name: "--preparation_workflow"
+        type: string
+        description: |
+          Run parameters override, preparation workflow. 
+      - name: --qc_only
+        type: boolean_true
+        description: |
+          Quickly generate run stats for single tile without generating FASTQ.
+          Use --include_tile/--exclude_tile to define custom tile set.
+      - name: --r1_cycles
+        type: integer
+        min: 1
+        description: |
+          Run parameters override, R1 cycles.
+      - name: --r2_cycles
+        type: integer
+        min: 1
+        description: |
+          Run parameters override, R2 cycles.
+      - name: "--split_lanes"
+        type: boolean_true
+        description: |
+          Split FASTQ files by lane.
+      # - name: "--skip_qc_report"
+      #   type: boolean_true
+      #   description: |
+      #     Do not generate HTML QC report.
+      - name: --strict
+        type: boolean_true
+        description: |
+          In strict mode any invalid or missing input file will terminate execution 
+          (overrides no_error_on_invalid and sets --error_on_missing)
+  
+        # --help, -h                      Display this usage statement              
+        # --input-remote, NAME            Rclone remote name for remote ANALYSIS_DIRECTORY
+        # --num-threads, -p NUMBER        Number of threads (default 1)
+        # --output-remote, NAME           Rclone remote name for remote OUTPUT_DIRECTORY
+        # --settings SELECTION            Run manifest settings override. This option may be specified multiple times.
```

Comment:
Not relevant? Or covered by the other arguments?

Reply:
Some are covered by other arguments, some by `meta` values. Some are just tricky/unclear to me what they do and/or are covered by nextflow.